notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Does this better belong at the top? Essentially cell #2 ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yes, it does!</p>